Solution

Barosz30 opened the pull request:
https://barosz30.github.io/react_todo-app-add-and-delete/

mgrinko committed:
fix deps

Patrykmclaren14 commented on src/components/TodoHeader.tsx:
```diff
@@ -0,0 +1,92 @@
+  return (
+    <header className="todoapp__header">
+      {/* this buttons is active only if there are some active todos */}
+      <button
+        type="button"
+        className="todoapp__toggle-all active"
+        data-cy="ToggleAllButton"
+      >
```
If your API doesn't have any todos, they should be hidden

patrykyo20 reviewed (approved):
Overall looks good!

mbruhler commented on src/components/TodosList.tsx:
```diff
@@ -0,0 +1,177 @@
+  );
+};
+
+// {/* This is a completed todo */}
```
Clear the comments

mbruhler reviewed (approved):
Works well, good job

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #12 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
